src/utils/get-ticket-no.ts

```diff
@@ -0,0 +1,26 @@
+export const IS_JIRA =
+  window.location.hostname.includes('atlassian.net') ||
+  window.location.pathname.startsWith('/browse/');
+
+export const IS_AZURE = window.location.hostname.includes('dev.azure.com');
+
+export const getTicketNo = () => {
+  const urlParams = new URLSearchParams(window.location.search);
+  const azureParam = urlParams.get('workitem');
+
+  const ticket = 'DEFAULT';
+
+  if (IS_JIRA) {
+    const jiraNo = window.location.href.match(/[A-Z]{2,}-\d+/);
+
+    if (jiraNo !== null) return jiraNo[0];
+  } else {
+    if (azureParam !== null) return `AZURE-${azureParam}`;
+
+    const azureNo = window.location.href.match(/edit\/(?<ticket>\d+)/);
+
+    if (azureNo !== null) return `AZURE-${azureNo.groups?.ticket}`;
```

Comment:
I wonder if we will have a case of same azure ticket number in multiple projects. I guess the same can happen in JIRA as well but it's an edge case so this should be good enough.

Reply:
Hm... Jira gets project name, like PERT-1, JOV-1, JAN-1... I guess this won't be a problem.
On the other hand, Azure gives us just ticket number, which will be saved as 'AZURE-${ticket=num}'
In this case, it can conflict with other projects.